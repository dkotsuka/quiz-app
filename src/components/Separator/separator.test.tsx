import React from 'react'
import { Separator } from '.'
import { create } from 'react-test-renderer';

describe("Separator", () => {
    const separator = create(<Separator width={10} height={10}/>)

    it("maches snapshot", () => {
        expect(separator).toMatchSnapshot()
    })
})