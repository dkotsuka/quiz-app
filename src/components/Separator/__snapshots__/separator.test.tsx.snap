// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator maches snapshot 1`] = `
<View
  height={10}
  style={
    Array [
      Object {
        "height": 10,
      },
    ]
  }
  width={10}
/>
`;
